pkg/spec/spec.go

```diff
@@ -544,6 +548,18 @@ func addIpcNS(config *CreateConfig, g *generate.Generator) error {
 	return nil
 }
 
+func addCgroupNS(config *CreateConfig, g *generate.Generator) error {
```

Comment:
I think we need a check in `getContainerCreateOptions()` to set `WithCgroupNsFrom()` for `container:$ID` CGroup mode